### PR TITLE
[ADD] website_sale_invoice_address: New module

### PR DESCRIPTION
**Pull request by chienandalu**
Since versión `12.0`, the e-Commerce orders are force to have the order partner
as the invoice address, wich is quite unconveniant for a B2B scenario.

This module restitutes the classic behavior.

cc @Tecnativa TT26408

### CODE REVIEW

File: website_sale_invoice_address/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2020 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Website Sale Invoice Address",
+    "summary": "Make the e-Commerce sale orders behave as in backend",
```
Comment (pedrobaeza) [inline website_sale_invoice_address/__manifest__.py]: This is too generic...

File: website_sale_invoice_address/models/website.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    def sale_get_order(
+        self,
+        force_create=False,
+        code=None,
+        update_pricelist=False,
+        force_pricelist=False,
+    ):
+        """Odoo commit 6e69de67071f2618e00856a74716d861b7f47391 breaks the old
+        flow than put the invoice partner of the commercial entity. In a B2B
```
Comment (pedrobaeza) [inline website_sale_invoice_address/models/website.py]:
```suggestion
        flow that puts the invoice partner of the commercial entity. In a B2B
```

**Review by pedrobaeza — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
:+1:

**Review by HaraldPanten — APPROVED** [review]
Functional review. Tested on runbot

**Comment by chienandalu** [comment]
This is ready

**Pull request merged**